examples/Demo/Shared/Components/ApiDocumentation.razor

```diff
@@ -49,7 +48,28 @@
 
                     }
                 </TemplateColumn>
-                <PropertyColumn Property="@(c => c.Default)" />
+
+@*                 <PropertyColumn Property="@(c => c.Default)" /> *@
```

Comment:
Please remove commented line (to not confuse people) and then I think we are good to go!